packages/server/lib/util/process_profiler.ts

```diff
@@ -208,7 +208,7 @@ export const _aggregateGroups = (processes: Process[]) => {
 export const _printGroupedProcesses = (groupTotals) => {
   const consoleBuffer = concatStream((buf) => {
     // get rid of trailing newline
-    debugVerbose(String(buf).trim())
+    debug(String(buf).trim())
```

Comment:
Are we intentionally updating this to debug rather than debugVerbose?

Reply:
Yes, we document this as being available with the non-verbose namespace: https://docs.cypress.io/guides/references/troubleshooting#Log-memory-and-CPU-usage